README.md

```diff
@@ -24,33 +29,81 @@ To deploy these nodes, download the jar from the releases tab on github
 ../web-container/webapps/openam/WEB-INF/lib directory where AM is deployed. Restart the web container to pick up the
 new nodes. The nodes will then appear in the authentication trees components palette.
 
-### Castle Profiler Node
-This node tags the AM login page with the Castle JS to collect information about the event.
+### Before you begin
 
-#### Castle Profiler Node Configuration
-* **App ID** - Castle App ID.
-* **Profiler URI** - Castle client side CDN link.
+1. Create a [Castle account](https://dashboard.castle.io/signup/new)
+2. Once you're signed in, go the the [Dashboard Settings page](https://dashboard.castle.io/settings/general) to configure the Castle Service with the APP ID and API Secret. Please not your Castle environment - you should use the "Sandbox" environment only for non-production use.
+3. We highly suggest you host our JS script yourself, otherwise it might get blocked by ad-blockers.
```

Comment:
Maybe a link to NPM (is that valid in this case) or link to documentation on how to host the JS in our docs.

Reply:
I added a link to our NPM package, but we'll have to update the README to make it clear how to host it yourself.